packages/terra-dialog/src/Dialog.jsx

```diff
@@ -37,6 +37,7 @@ const defaultProps = {
 const Dialog = ({ header, footer, onClose, children, ...customProps }) => {
   const dialogClassNames = cx([
     'dialog',
+    customProps.className,
```

Comment:
This change will also need to reverse the spread order on this line: https://github.com/cerner/terra-core/blob/63f7ece5df3d563a623edff83a79054074264c7d/packages/terra-dialog/src/Dialog.jsx#L48

Otherwise custom classNames will always overwrite dialogClassNames

Reply:
Thanks for the eyes!

https://github.com/cerner/terra-core/pull/1267/commits/82abfaa85715e73c13b3084555a4c387362cd65f